spago.yaml

```diff
@@ -1,5 +1,6 @@
 package:
   name: purescm
+  version: 1.0.1-pre.0
```

Comment:
I am surprised this parses, it should not 😄

Reply:
Should I change the versioning scheme for prereleases? For example just stick with bumping the minor version.

Reply:
Yeah, it does not - how is CI passing??

<img width="504" alt="image" src="https://github.com/purescm/purescm/assets/5480361/3d8f0eae-e82a-4d75-b253-ec331aeb32e4">

Reply:
It does work locally for me as well with spago `0.93.14`

Reply:
Ah I see - the version field needs to be inside a `publish` block. So right now the version that is returned by the `BuildInfo` should be 0.0.0

Reply:
```suggestion
  publish:
    version: 1.0.1
```

Reply:
So should we then stick with bumping the patch version for prereleases or is there another versioning that spago supports for prereleases?

Reply:
No support for prereleases. I think we are open to discussing adding support, but it should happen [in the registry first](https://github.com/purescript/registry-dev)

Reply:
SemVer is terrible at prereleases anyways, so we'd need to diverge from it I think

Reply:
Ok, I removed the prerelease script and using `1.0.1` for the next one.